src/components/UserTable.tsx

```diff
@@ -435,12 +436,22 @@ const UserTable: React.FC<UserTableProps> = ({
       setLoading(true);
       try {
         const fields = ["age", "districts", "states", "blocks", "gender"];
-        const limit = pageLimit;
-        const offset = pageOffset * limit;
+        let limit = pageLimit;
+        let offset = pageOffset * limit;
         // const filters = { role: role , status:"active"};
         const sort = sortBy;
         console.log("filters", filters);
-        const resp = await userList({ limit, filters, sort, offset, fields });
+       let resp;
+        if(filters.name)
+        {
+          offset=0;
+           resp = await userList({ limit, filters, sort, offset, fields });
```

Comment:
get resp = await userlist line out of if and else condition as there is no difference in it